### PR TITLE
Removed testgrid links for azure and openstack cluster < 1.15

### PR DESCRIPTION
**Pull request by schrodit**
Removed the testgrid links for Azure and Openstack clusters with K8s version < 1.15 and added a disclaimer

**How to categorize this PR?**

/area Documentation
/priority small

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```improvement operator
NONE
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,12 +35,14 @@ Conformance test results of latest stable Gardener release, transparently visibl
...
 | **Packet** | N/A | N/A | N/A | N/A | N/A | N/A | N/A | N/A | N/A |
 
+[1] Version is technically supported but not activly tested. Therefore, the usage of this version is at your own risk.
```
Comment (dguendisch) [inline README.md]: Would add `no longer` to indicate that we tested them in the past but don't see value for testing anymore.
Would rephrase the second sentence as everything in public gardener is used at "your own risk".
How about that?
```suggestion
[1] Version is technically supported but no longer actively tested. Regressions will go unnoticed.
```

**Review by dguendisch — CHANGES_REQUESTED** [review]
After this PR please also update https://github.com/gardener/gardener-extension-provider-azure

**Review by rfranzke — APPROVED** [review]

**Review by vpnachev — APPROVED** [review]
/lgtm

**Review by dguendisch — APPROVED** [review]
